dpgen2/constants.py

```diff
@@ -3,6 +3,7 @@
 train_script_name = "input.json"
 train_log_name = "train.log"
 model_name_pattern = "model.%03d.pb"
+model_name_match_pattern = "model.*pb"
```

Comment:
I suggest a more strict pattern:

```suggestion
model_name_match_pattern = r"model\.[0-9]{3,}\.pb"
```